src/hooks/useTable.js

```diff
@@ -139,14 +139,16 @@ export const useTable = (props, ...plugins) => {
   // This function is absolutely necessary and MUST be called on
   // any rows the user wishes to be displayed.
   api.prepareRow = row => {
-    const { index } = row
+    const { path, index } = row
     row.getRowProps = props =>
       mergeProps(
-        { key: ['row', index].join('_') },
+        { key: ['row', path || index].join('_') },
```

Comment:
Should we just use `path` all the time? Is `path` set when not using `useExpanded`?